### PR TITLE
version 0.4.1

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
initial version 0.4.1

[upstream ](https://github.com/cbrunet/python-poppler/tree/v0.4.1)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+source:
+  url: https://files.pythonhosted.org/packages/15/ff/e9c8f176c376223146d212771016595b4ad8d0a83d40636b7b39798a7219/python_poppler-{{ version }}.tar.gz
+  sha256: e6ca5c23ec02350bf2cef85a6bf9f1b261796436db478176f9d24fb14eeecc6a
+
+build:
+  number: 0
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  ignore_run_exports:
+    - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Do we want to ignore_run_exports for python?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - poppler
+
+  requires:
+    - pip
+  commands:
+    - pip check # [not win]
+
+about:
+  home: https://github.com/cbrunet/python-poppler
+  license: GPL-2.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  license: GPL-2.0-or-later
```

File: abs.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#upload_channels:
+#  - sfe1ed40
+
+channels:
+  - https://staging.continuum.io/prefect/fs/poppler-feedstock/pr16/f8c28aa
```
Comment (skupr-anaconda) [inline abs.yaml]: To be removed before merging and enabling:
```
upload_channels:
  - sfe1ed40
```

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'Python binding to the poppler-cpp library'
+  description: |
+    python-poppler is a Python binding to the poppler-cpp library. It allows to read, render, or modify PDF documents. More specifically, it currently allows to
+    read an modify document meta data, list and read embedded documents, list the fonts used by the document, search or extract text on a given page of the document,
+    render a page to a raw image, get info about transitions effects between the pages, read the table of contents of the document.
+  doc_url: https://github.com/cbrunet/python-poppler
+  dev_url: https://github.com/python-rapidjson/python-rapidjson
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  dev_url: https://github.com/cbrunet/python-poppler
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    - cmake >=3.18
+    - ninja       # [win]
+    - pkg-config
+    
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - meson >=1.0.0
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - meson >=1.1.0
```
https://github.com/cbrunet/python-poppler/blob/v0.4.1/meson.build#L7

File: recipe/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    - meson-python 
+    - poppler 22.12.0
```
Comment (cbouss) [inline recipe/meta.yaml:31]: You should unvendor pybind and add it as a host dependency.
https://github.com/cbrunet/python-poppler/blob/v0.4.1/meson.build#L16

File: recipe/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+build:
+  number: 0
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv # [unix]
+  
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake >=3.18
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
No cmake involved here, this is a meson build.

File: recipe/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    - ninja       # [win]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+    - poppler
+
+test:
+  imports:
+    - poppler
+
+  requires:
+    - pip
+  commands:
+    - pip check # [not win]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - pip check
```
Or what is the reason for commenting pip check here?

**Review by cbouss — CHANGES_REQUESTED** [review]

**Review by cbouss — APPROVED** [review]

**Pull request merged**